problems/loops.js

```diff
@@ -143,7 +234,32 @@ function fizzBuzzPart1() {}
  * @returns {Array}
 
  */
-function fizzBuzzPart2() {}
+const fizzBuzzPart2 = ()  =>{
+  let array = [];
+  let i = 1;
+  while(i <= 100){
+    if (i % 5 === 0 && i % 3 === 0){
+      array.push("FizzBuzz")
+      i++;
+    }
+    else if (i % 3 === 0){
+     array.push ("Fizz")
+     i++
+  }
+    else if (i % 5 === 0){
```

Comment:
put else if on same line as previous closing } . 
Use prettier to fix your code.